OpenSim/Examples/Plugins/VolumetricContactExample/VolumetricContactForce.h

```diff
@@ -0,0 +1,134 @@
+#ifndef _OPENSIM_VOLUMETRIC_CONTACT_FORCE_PLUGIN_H_
```

Comment:
Update this to be _OPENSIM_<YOUR_CLASS_NAME> once you decide what you want to name it. _PLUGIN_ is unnecessary.